dist/pf-relative.js

```diff
@@ -12,6 +12,14 @@ angular.module('pf-relative', [])
     var relative = new RelativeDate(scope.date);
     element.attr('datetime', scope.date);
     relative.onTick(element.text.bind(element));
+
+    scope.$watch('date', function (newDate) {
```

Comment:
How does it behave if we get rid of the anonymous function and just go with the following?
```javascript
scope.$watch('date', relative.setDate);
```
I think it might break because `this` will be `window`, but if it works, I think that's a better approach.
And perhaps, because we already have `scope.date`, I guess replacing the string `'date'` with `scope.date` works too.